api4/config.go

```diff
@@ -407,6 +407,12 @@ func migrateConfig(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	isCloud := c.App.License() != nil && *c.App.License().Features.Cloud
+	if isCloud {
+		c.Err = model.NewAppError("migrateConfig", "api.config.migrate_config_restricted.app_error", nil, "", http.StatusMethodNotAllowed)
```

Comment:
Suggestion: Ideally `http.StatusForbidden` should be used instead of `http.StatusMethodNotAllowed` since the API is restricted/forbidden to be used.

Reply:
Yeah, makes sense.